packages/presentation/lib/widgets/image_widget.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/const.dart';
+
+class ImageWidget extends StatelessWidget {
+  final String? _imageUrl;
+
+  const ImageWidget(
+    this._imageUrl, {
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return _imageUrl == null
+        ? Text(
+            'Image Not Exist Yet',
+            style: ImageWidgetStyles.imageNotExistStyle,
+          )
+        : Image.network(
+            _imageUrl!,
```

Comment:
seems that u have no need in force-unwrap

Reply:
I think no, network needs not null string.

Reply:
hm, strange. the check u have higher should be enough for dart compiler:

_imageUrl == null

but if it is not compiled without force-unwrap - keep it